app/src/main/java/com/sdp/movemeet/view/home/HomeScreenActivity.java

```diff
@@ -33,11 +33,6 @@ protected void onCreate(Bundle savedInstanceState) {
         if (ActivityCompat.checkSelfPermission(this, Manifest.permission.ACCESS_FINE_LOCATION) != PackageManager.PERMISSION_GRANTED) {
             ActivityCompat.requestPermissions(this, new String[]{Manifest.permission.ACCESS_FINE_LOCATION}, REQUEST_CODE);
         }
-
```

Comment:
Yes we really have to remove useless code

Reply:
Yep, we will have to do a last round of check, to make sure that there are no more of these.